### DISCUSSION TITLE
diyHue 2.0.0

### DISCUSSION THREAD

**Discussion by legovaer** [diyhue/diyHue#925] [General]
After having several conversations with the maintainers of the diyHue project we have agreed that the current codebase of the diyHue project is quite complex. This makes it really difficult for people willing to contribute. As the owner of the project is quite busy with real-life projects, we have agreed that I will start working on a complete new version of diyHue. 

Obviously, I am welcoming everyone to help me out with this. 

The milestone for the first release of this new version is `2.0.0`. All code for this new version will be stored in the `next-release` branch. Issues related to this new version can be labelled with `v2`.

As I don't want to re-invent the wheel, I will re-use code from various existing projects like `aiohue`.

**Comment by YellowNest** [comment]
Is there any progress?
